src/ImageSharp/Formats/Tiff/TiffDecoderOptionsParser.cs

```diff
@@ -69,7 +69,7 @@ public static void VerifyAndParse(this TiffDecoderCore options, ExifProfile exif
             options.Predictor = frameMetadata.Predictor ?? TiffPredictor.None;
             options.PhotometricInterpretation = frameMetadata.PhotometricInterpretation ?? TiffPhotometricInterpretation.Rgb;
             options.BitsPerPixel = frameMetadata.BitsPerPixel != null ? (int)frameMetadata.BitsPerPixel.Value : (int)TiffBitsPerPixel.Bit24;
-            options.BitsPerSample = GetBitsPerSample(frameMetadata.BitsPerPixel);
+            options.BitsPerSample = frameMetadata.BitsPerSample != null ? frameMetadata.BitsPerSample?.ToArray() : Array.Empty<ushort>();
```

Comment:
I'm curious, why are we using `ToArray()` here? the only use case for that I would have thought would have been writing to the EXIF tag when encoding.

Reply:
BitsPerSample was still a `ushort[]`, I have changed that now to the TiffBitsPerSample struct